libpimeval/src/pimParamsPerf.cpp

```diff
@@ -530,33 +530,40 @@ pimParamsPerf::s_bitsimdPerfTable = {
   }},
 };
 
-//! @brief  pimParamsPerf ctor
+//! @brief  pimParamsPerf ctortodo
 pimParamsPerf::pimParamsPerf(pimParamsDram* paramsDram)
   : m_paramsDram(paramsDram)
 {
   m_tR = m_paramsDram->getNsRowRead() / 1000000.0;
   m_tW = m_paramsDram->getNsRowWrite() / 1000000.0;
   m_tL = m_paramsDram->getNsTCCD() / 1000000.0;
+  m_eR = m_paramsDram->getPjRowRead() / 1000000000.0; // Convert pJ to mJ
+  m_eL = m_paramsDram->getPjLogic() / 1000000000.0; // Convert pJ to mJ
```

Comment:
Looks like this m_eL is the energy of one BSLU. To compute the energy of a bit-serial SIMD logic operation, we need the following formula: 
m_eL * numColsPerSubarray * numActiveSubarrays * numPasses